server/routes/topic_page.py

```diff
@@ -36,15 +40,20 @@ def topic_page(topic_id=None, place_dcid=None):
     return flask.render_template('topic_page_landing.html')
 
   all_configs = current_app.config['TOPIC_PAGE_CONFIG']
-  if os.environ.get('FLASK_ENV') == 'local':
+  if g.env == 'local':
     all_configs = libutil.get_topic_page_config()
   topic_configs = all_configs.get(topic_id, [])
-  if len(topic_configs) < 1:
-    return "Error: no config found"
 
-  if topic_id == 'dev' and os.environ.get('FLASK_ENV') == 'production':
+  if topic_id in _DEBUG_TOPICS:
+    if g.env in ['local', 'autopush', 'dev']:
+      if topic_id == _NL_DISASTER_TOPIC:
+        topic_configs = [libutil.get_nl_disaster_config()]
+  else:
```

Comment:
I think this else statement might be in the wrong place? If I'm not mistaken, this is saying `if not topic_id in _DEBUG_TOPICS: flask.abort(404)`

Reply:
thanks for catching that!